en/contributing/plugins.md

```diff
@@ -0,0 +1,102 @@
+# Writing Plugins

```

Comment:
@julianoes OK, new doc on writing "standard" plugins. I'm still not completely happy with it. What it tries to capture is:
- Plugins all have same location - subfolder of /plugins
- Plugins all have same structure - with implementation etc. This structure is shown by the external example
- If both above are true for a plugin it will be auto added to DroneCore at build time. 
- Plugins are also defined for SDKs 
- Integration tests should be created for plugins
- Example code should be created too, but after tests.

Reply:
I'm not sure I would mention the "SDK" thing.

The rest is exactly true.

Reply:
I have removed. everywhere.